api/src/v2/Registration/Assembler/SiriusToLayDeputyshipDtoAssembler.php

```diff
@@ -87,6 +87,10 @@ private function determineReportTypeIsSupported(?string $reportType)
             default => false
         };
 
-        return $supported ? $reportType : '';
```

Comment:
Maybe it's the Go side of me but I believe writing something like:
```
if (!$supported) {
    throw new InvalidArgumentException();
}

return $reportType
```

Feels cleaner (to me anyways), as from a Go perspective you're taught to deal with any errors you may potentially have first and then if you have none you just return what's expected. 

Although I know this is not Go I still feel like that way of thinking feels more natural to me. Check for an error first -> handle the error if there is one -> if no error return what we're expecting 

This way you also make the code a little cleaner as there's no `else` statement :)